eng/common/TestResources/New-TestResources.ps1

```diff
@@ -480,6 +479,30 @@ Name of the cloud environment. The default is the Azure Public Cloud
 .PARAMETER AdditionalParameters
 Optional key-value pairs of parameters to pass to the ARM template(s).
 
+.PARAMETER SubscriptionConfiguration
+Name of a subscription configuration secret in a Key Vault. Stored as a JSON
```

Comment:
This should be the exact configuration and not just the name. We should be letting devops get the value from keyvault for us.